docs/config.toml

```diff
@@ -24,13 +24,19 @@ defaultContentLanguage = "any"
     weight = 1
     [languages.node.params]
       icon = "node-js"
+      frameworks = [
+        ["Express", "Express is a minimal and flexible Node.js web application framework that provides a robust set of features for web and mobile applications."]
+      ]
 
   [languages.python]
     languageName = "Python"
     contentDir = "content/python"
     weight = 2
     [languages.python.params]
       icon = "python"
+      frameworks = [
```

Comment:
Could we rename this to `request_frameworks`?

Reply:
I also don't think descriptions are necessary for frameworks that we don't integrate with.